### DISCUSSION TITLE
Stable version

### DISCUSSION THREAD

**Discussion by BabyOilJohnson** [ImranR98/Obtainium#807] [Q&A] [answered]
When will Obtainium be out of beta?

**Comment by ImranR98** [comment, marked as answer]
Now that #25 is closed, I want to clean up the code a bit (really messy right now, focused more on getting things working than maintainability). Will probably release a 1.0 after that since the number of bugs doesn't seem to be that high/severe recently. But cleaning things up is boring so I'm not sure when it will happen 😅.
